peer.go

```diff
@@ -1728,9 +1728,11 @@ func (p *peer) ChannelSnapshots() []*channeldb.ChannelSnapshot {
 	return snapshots
 }
 
-// genDeliveryScript returns a new script to be used to send our funds to in
+// GenDeliveryScript returns a new script to be used to send our funds to in
 // the case of a cooperative channel close negotiation.
-func (p *peer) genDeliveryScript() ([]byte, error) {
+//
+// NOTE: Part of the lnpeer.Peer interface.
+func (p *peer) GenDeliveryScript() ([]byte, error) {
```

Comment:
hm, not sure this should be part of the `Peer` interface.

Reply:
yeah, it's a bit of a weird one. I'll see if I can thread it down another way :)